frontend/src/metabase/components/MetadataInfo/Popover/Popover.tsx

```diff
@@ -82,9 +78,6 @@ export function Popover({
         onMouseUp={stopPropagation}
         className={group.shouldDelay ? Animation.fadeIn : null}
       >
-        {/* HACK: adds an element between the target and the card */}
```

Comment:
This hack was introduced in #38433 to fix this issue: https://github.com/metabase/metabase/pull/38433/files#r1478402947

Removing the hack does not bring the issue back (I guess because we have better popover close delay now).